applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BROADCAST_SOURCE_H_
+#define _BROADCAST_SOURCE_H_
+
+#include "le_audio.h"
+#include "broadcast.h"
+
+/**
+ * @brief	Get the data to advertise.
+ *
+ * @param[out]	ext_adv		Pointer to the pointer of bt_data used for extended advertising.
+ * @param[out]	ext_adv_size	Pointer to size of @p ext_adv.
+ * @param[out]	per_adv		Pointer to the pointer of bt_data used for periodic advertising.
+ * @param[out]	per_adv_size	Pointer to size of @p per_adv.
+ */
+void broadcast_source_adv_get(const struct bt_data **ext_adv, size_t *ext_adv_size,
+			      const struct bt_data **per_adv, size_t *per_adv_size);
+
+/**
+ * @brief	Start the Bluetooth LE Audio broadcast (BIS) source.
+ *
+ * @param[in]	ext_adv		Pointer to the extended advertising set, can be NULL if a stream
+ *				is restarted.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int broadcast_source_start(struct bt_le_ext_adv *ext_adv);
+
+/**
+ * @brief	Stop the Bluetooth LE Audio broadcast (BIS) source.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int broadcast_source_stop(void);
+
+/**
+ * @brief	Broadcast the Bluetooth LE Audio data.
+ *
+ * @param[in]	enc_audio	Encoded audio struct.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int broadcast_source_send(struct encoded_audio enc_audio);
+
+/**
+ * @brief	Enable the LE Audio broadcast (BIS) source.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int broadcast_source_enable(void);
```

Comment:
Remove. It seems this got messed up in a rebase in https://github.com/alexsven/sdk-nrf/pull/25/files#diff-f960e1ec977a994f52a902c0645a161e09aae73315c58ac9604efa66a04ca6f4R51-R56